hail_search/web_app.py

```diff
@@ -116,6 +121,7 @@ async def init_web_app():
     app = web.Application(middlewares=[error_middleware], client_max_size=(1024**2)*10)
     app.add_routes([
         web.get('/status', status),
+        web.post('/reload_globals', reload_globals),
```

Comment:
The internet convinced me that `POST` was more correct than `GET` for this, since we're potentially modifying server state.  It's a gray area though.